ui/app/AppLayouts/Profile/ProfileLayout.qml

```diff
@@ -155,13 +157,51 @@ StatusSectionLayout {
             }
 
             AppearanceView {
+                id: appearanceView
                 implicitWidth: parent.width
                 implicitHeight: parent.height
-
-                appearanceStore: root.store.appearanceStore
                 sectionTitle: root.store.getNameForSubsection(Constants.settingsSubsection.appearance)
                 contentWidth: d.contentWidth
-                systemPalette: root.systemPalette
+
+                currentFontSize: localAccountSensitiveSettings.fontSize
```

Comment:
Shouldn't this `localAccountSensitiveSettings.fontSize` also got thru some store?

Reply:
I just moved 'up' what the component was doing (which ensures it still works as expected), we can/should refactor some of this logic later once the settings themselves are moved into storybook, but for now doing so could risk introducing regressions